sailfish-py/gateway/gateway.py

```diff
@@ -100,9 +103,27 @@ def post_job():
     print("Received Job:")
     print(job)
 
+    while not check_amq_broker(host,port):
```

Comment:
I know this is just an example but can't we cater a time-out here? cluster conditions can results in a pending broker pod..

Reply:
Since this is by rest call, I'll test when/if it times out by that already. Maybe we can just configure Knative to timeout http requests after ~2min

Reply:
It currently times out after ~600s, the knative pod also terminates. The caller gets a timeout error so that's detectable.